tests/tests_integration/test_api/test_datapoints.py

```diff
@@ -2681,6 +2681,33 @@ def test_include_granularity_name_fails(self, cognite_client, one_mill_dps_ts):
                 id=one_mill_dps_ts[0].id, limit=5, granularity="1d", aggregates="min", include_granularity_name=None
             )
 
+    def test_include_unit(
```

Comment:
I would like a parametrize for include_status True/False as well since that logic seems to be touched